config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_apikeyskeys.apikeys.cnrm.cloud.google.com.yaml

```diff
@@ -0,0 +1,249 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Hi Justin, is the generation of this CRD purely manual? Or is there some automation involved?

Reply:
It uses the "classic" generation mechanism (i.e. not automated generation) from a terraform resource